mule-extensions-api-persistence/src/test/java/org/mule/runtime/extension/api/persistence/LoadAndSerializeXmlExtensionModelsTestCase.java

```diff
@@ -116,7 +115,9 @@ public void serializeDeserializeExternalModel() throws IOException {
   public void serializeDeserializeMock() throws IOException {
 
     final String serializedExtensionModelString = extensionModelJsonSerializer.serialize(originalExtensionModel);
-    FileUtils.writeStringToFile(new java.io.File(SERIALIZED_EXTENSION_MODEL_STRING_JSON), serializedExtensionModelString);
+
+    FileUtils.writeStringToFile(createTempFile(SERIALIZED_EXTENSION_MODEL_STRING_JSON, ".tmp").toFile(),
```

Comment:
I don't know, I just improved the test case to don't mess up the project with the auto generated files.
@alegmarra ?